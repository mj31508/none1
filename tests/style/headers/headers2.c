int func0(void);

typedef unsigned int sample;

struct sample_s
{
	int data;
};

enum sample_e
{
	FIRST = 0,
	SECOND
};

union sample_u
{
	char rgba[4];
	int value;
};
