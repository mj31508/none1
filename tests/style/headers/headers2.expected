tests/style/headers/headers2.c:1: WARNING: externs should be avoided in .c files
tests/style/headers/headers2.c:2: WARNING: externs should be avoided in .c files
tests/style/headers/headers2.c:4: WARNING: typedefs should be avoided in .c files
tests/style/headers/headers2.c:6: WARNING: struct definition should be avoided in .c files
tests/style/headers/headers2.c:11: WARNING: enum definition should be avoided in .c files
tests/style/headers/headers2.c:17: WARNING: union definition should be avoided in .c files
total: 0 errors, 6 warnings, 32 lines checked
